app/assets/javascripts/jquery.mb.YTPlayer.js

```diff
@@ -0,0 +1,9 @@

```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'ytp' was used before it was defined.<br>Expected '{' and instead saw 'jQuery'.<br>Expected '{' and instead saw '-'.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>Missing '()' invoking a constructor.<br>Expected '{' and instead saw 'h'.<br>Expected '{' and instead saw 'for'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>Creating global 'for' variable. Should be 'for (var n ...'.<br>Expected '{' and instead saw 'k'.<br>'k' used out of scope.<br>Too many errors. (80% scanned).